workflows/eager.nf

```diff
@@ -560,6 +561,35 @@ workflow EAGER {
         ch_multiqc_files = ch_multiqc_files.mix(GENOTYPE.out.mqc.collect { it[1] }.ifEmpty([]))
     }
 
+    //
+    // SUBWORKFLOW: Run mtDNA Haplogroup Classification
+    //
+
+    if (params.run_mtdna_haplogroup) {
+        if (!params.run_genotyping) {
+            error "Cannot run mtDNA haplogroup classification (--run_mtdna_haplogroup) without running genotyping (--run_genotyping). VCF files are required as input."
```

Comment:
So there used to be a vcf2genome in eager2 that output a consensus fasta. Is that still a thing? Because haplogrep3 also works with mtDNA sequences aligned to rCRS or RSRS.

Reply:
no. vcf2genome is discontinued and will not be supported in eager 3.* .
Consensus sequence calling will be added soon™ ( #1142 ).